versions/3.0.4.md

```diff
@@ -126,11 +126,11 @@ The OpenAPI Specification is versioned using a `major`.`minor`.`patch` versionin
 
 Occasionally, non-backwards compatible changes may be made in `minor` versions of the OAS where impact is believed to be low relative to the benefit provided.
 
-An OpenAPI document compatible with OAS 3.\*.\* contains a required [`openapi`](#oasVersion) field which designates the version of the OAS that it uses.
+An OpenAPI description document compatible with OAS 3.\*.\* contains a required [`openapi`](#oasVersion) field which designates the version of the OAS that it uses.
```

Comment:
Do we really need to add the "description" qualifier here? Is there some other kind of "OpenAPI document" that we are trying to distinguish from?

Reply:
I think it does help in this context, so I think we should leave this as-is